unlock-app/src/components/legacy-auth/LegacyAuthWrapper.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useContext, useMemo } from 'react'
+import { Web3Service } from '@unlock-protocol/unlock-js'
+import { Web3ServiceContext } from '../../utils/withWeb3Service'
+import { WalletServiceContext } from '../../utils/withWalletService'
+import { AuthenticationContext } from '../../contexts/AuthenticationContext'
+import { ConfigContext } from '../../utils/withConfig'
+import { SIWEProvider } from '~/hooks/useSIWE'
+import { networks } from '@unlock-protocol/networks'
+import { useLegacyProvider } from '~/hooks/useLegacyProvider'
+
+const Web3ServiceProvider = Web3ServiceContext.Provider
+
+interface ProvidersProps {
+  children: any
+}
+/**
+ * Utility providers set to retrieve content based on network settings
+ * @returns
+ */
+const Providers = ({ children }: ProvidersProps) => {
+  // TODO: remove this and the Web3ServiceProvider
+  const web3Service = useMemo(() => {
+    return new Web3Service(networks)
```

Comment:
We probably don't need this. do we? Let's clean up in a later PR

Reply:
yeah exactly